ci/Dockerfile

```diff
@@ -23,29 +23,42 @@
 ARG CUDA_VERSION=11.8.0
 ARG OS_RELEASE=8
 ARG TARGETPLATFORM=linux/amd64
+
 # multi-platform build with: docker buildx build --platform linux/arm64,linux/amd64 <ARGS> on either amd64 or arm64 host
 # check available official arm-based docker images at https://hub.docker.com/r/nvidia/cuda/tags (OS/ARCH)
 FROM --platform=$TARGETPLATFORM nvidia/cuda:$CUDA_VERSION-devel-rockylinux$OS_RELEASE
+
+### Software versions for development
 ARG TOOLSET_VERSION=11
+ARG CMAKE_VERSION=3.30.4
```

Comment:
nit: some comment about the version choice sine I recalls there is always some caveat why a particular version is used

Reply:
Will do that in the follow up work.